frontend/src/pages/Landing/Landing.tsx

```diff
@@ -14,6 +14,7 @@ export const Landing = ({ children }: { children: ReactNode }) => {
 
 	return (
 		<Box
+			backgroundColor="p12"
```

Comment:
As per the design, shared by @julian-highlight, the background was supposed to be whatever is going in the app behind as should've been a modal, hence I used the transparent background to place this on top of the routes while intercepting.